backend/src/main.py

```diff
@@ -0,0 +1,48 @@
+from typing import List
+from fastapi import FastAPI, Depends
+from sqlmodel import Session, select
+
+from .model import Artist, Album, Genre, Track
+from .db import init_db, get_session
+
+class Song():
```

Comment:
why ?

Reply:
Ok, I'm waiting this commit before merge :)

Reply:
On it

Reply:
Perfect